packages/xo-web/src/xo-app/backup/new/index.js

```diff
@@ -1150,28 +1137,20 @@ export default decorate([
                               </Tooltip>{' '}
                               <input
                                 checked={offlineBackup}
+                                disabled={offlineSnapshot || checkpointSnapshot}
                                 onChange={effects.setOfflineBackup}
                                 type='checkbox'
                               />
                             </label>
                           </FormGroup>
                         </div>
                       )}
-                      {!state.offlineBackupActive && (
-                        <FormGroup>
-                          <label>
-                            <strong>{_('offlineSnapshot')}</strong>{' '}
-                            <Tooltip content={_('offlineSnapshotInfo')}>
-                              <Icon icon='info' />
-                            </Tooltip>{' '}
-                            <input
-                              checked={offlineSnapshot}
-                              onChange={effects.setOfflineSnapshot}
-                              type='checkbox'
-                            />
-                          </label>
-                        </FormGroup>
-                      )}
+                      <SnapshotModeSelect
+                        checkpointSnapshot={checkpointSnapshot}
+                        disabled={state.offlineBackupActive}
+                        offlineSnapshot={offlineSnapshot}
+                        setGlobalSettings={effects.setGlobalSettings}
```

Comment:
onChange?

Reply:
Better let an explicit name to know what this function change, because it not change the snapshot mode but it set the global settings.